content/en/blog/2021/better-external-authz/index.md

```diff
@@ -0,0 +1,545 @@
+---
+title: Better External Authorization
+subtitle: Integrate external authorization system (e.g. OPA, oauth2-proxy, etc.) with Istio using AuthorizationPolicy
+description: AuthorizationPolicy now supports CUSTOM action to delegate the authorization to external system.
+publishdate: 2021-02-09
+attribution: Yangmin Zhu (Google), Craig Box (Google) and Christian Posta (Solo.io)
```

Comment:
I'd be happy with a "Thanks to Craig Box and Christian Posta for reviewing drafts of this blog" at the end.  I don't feel I deserve attribution.  @christian-posta thoughts?